.env.production

```
@@ -50,4 +51,8 @@ GOOGLE_REDIRECT_URI=${GOOGLE_REDIRECT_URI}
 FRONTEND_URL=${FRONTEND_URL}
 
 # 로그인 리다이렉트 URL
-FRONTEND_LOGIN_CALLBACK_URL=${FRONTEND_LOGIN_CALLBACK_URL}
\ No newline at end of file
+FRONTEND_LOGIN_CALLBACK_URL=${FRONTEND_LOGIN_CALLBACK_URL}# Debug token issuance 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)